client/src/pages/tasks/Show.js

```diff
@@ -98,6 +99,12 @@ export default class TaskShow extends Page {
 							this.renderOrg()
 						}
 
+						{task.parentTask && task.parentTask.id &&
```

Comment:
I suggest to make this customizable in the dictionary and update anet-schema.yml